scapy/layers/ntp.py

```diff
@@ -394,6 +394,9 @@ def getfield(self, pkt, s):
             extensions_len = len(extensions)
             while extensions_len >= 16:
                 ext_len = struct.unpack("!H", extensions[2:4])[0]
+                ext_len = min(ext_len, extensions_len)
+                if ext_len < 1:
+                    ext_len = extensions_len
```

Comment:
What special cases are you fixing here? Maybe it could be good to add one in the regression tests?